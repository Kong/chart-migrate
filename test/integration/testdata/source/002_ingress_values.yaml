deployment:
  test:
    enabled: false

controller:
  proxy:
    # this should _not_ carry over into the migrated values.yaml
    nameOverride: "{{ .Release.Name }}-gateway-proxy"

  enabled: true

  deployment:
    kong:
      enabled: false

  ingressController:
    enabled: true
    image:
      repository: kong/kubernetes-ingress-controller
      tag: "3.0"
    args:
    - --dump-config
    - "true"
  
    env:
      kong_admin_tls_skip_verify: true
      election_id: palmtree
    customEnv:
      TZ: "Europe/Berlin"
      FAKE: "whatever"
  
    livenessProbe:
      httpGet:
        path: "/healthz"
        port: 10254
        scheme: HTTP
      initialDelaySeconds: 5
      timeoutSeconds: 5
      periodSeconds: 10
      successThreshold: 1
      failureThreshold: 3
    readinessProbe:
      httpGet:
        path: "/readyz"
        port: 10254
        scheme: HTTP
      initialDelaySeconds: 5
      timeoutSeconds: 5
      periodSeconds: 10
      successThreshold: 1
      failureThreshold: 3
    resources:
      limits:
        cpu: 100m
        memory: 256Mi
      requests:
        cpu: 50m
        memory: 128Mi

    serviceAccount: "my-account"
    hostNetwork: true
    tmpDir:
      sizeLimit: 1Gi
    watchNamespaces:
    - default
    - other
    ingressClass: "kong"
    ingressClassAnnoations:
      example.com/foo: foo
      example.com/bar: bar
    rbac:
      create: true
    konnect:
      runtimeGroupID: abc123

    adminApi:
      tls:
        client:
          enabled: true


    gatewayDiscovery:
      enabled: true
      generateAdminApiService: true

  terminationGracePeriodSeconds: 111
  tolerations:
    fakeNonsense: "idk the actual format"
  nodeSelector: "whatever"
  priorityClassName: "whatever"

  podAnnotations:
    kuma.io/gateway: enabled
    # This port must match your Kong admin API port. 8444 is the default.
    # If you set gateway.admin.tls.containerPort, change these annotations
    # to use that value.
    traffic.kuma.io/exclude-outbound-ports: "8444"
    traffic.sidecar.istio.io/excludeOutboundPorts: "8444"

  podLabels:
    example.com/foo: foo
    example.com/bar: bar

  deploymentAnnotations:
    example.com/foo: foo
    example.com/bar: bar

  replicaCount: 3

  securityContext:
    something: "whatever"

  containerSecurityContext:
    readOnlyRootFilesystem: false

gateway:
  enabled: true
  deployment:
    kong:
      enabled: true

  admin:
    enabled: true
    type: ClusterIP
    clusterIP: None

  ingressController:
    enabled: false

  env:
    role: traditional
    database: "off"
