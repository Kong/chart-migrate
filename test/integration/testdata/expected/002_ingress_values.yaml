admin:
  clusterIP: None
  enabled: true
  type: ClusterIP
deployment:
  kong:
    enabled: true
env:
  database: "off"
  role: traditional
ingressController:
  adminApi:
    tls:
      client:
        enabled: true
  deployment:
    annotations:
      example.com/bar: bar
      example.com/foo: foo
    pod:
      annotations:
        kuma.io/gateway: enabled
        traffic.kuma.io/exclude-outbound-ports: "8444"
        traffic.sidecar.istio.io/excludeOutboundPorts: "8444"
      container:
        args:
        - --dump-config
        - "true"
        customEnv:
          FAKE: whatever
          TZ: Europe/Berlin
        env:
          election_id: palmtree
          kong_admin_tls_skip_verify: true
        image:
          repository: kong/kubernetes-ingress-controller
          tag: "3.0"
        livenessProbe:
          failureThreshold: 3
          httpGet:
            path: /healthz
            port: 10254
            scheme: HTTP
          initialDelaySeconds: 5
          periodSeconds: 10
          successThreshold: 1
          timeoutSeconds: 5
        readinessProbe:
          failureThreshold: 3
          httpGet:
            path: /readyz
            port: 10254
            scheme: HTTP
          initialDelaySeconds: 5
          periodSeconds: 10
          successThreshold: 1
          timeoutSeconds: 5
        resources:
          limits:
            cpu: 100m
            memory: 256Mi
          requests:
            cpu: 50m
            memory: 128Mi
        securityContext:
          readOnlyRootFilesystem: false
      labels:
        example.com/bar: bar
        example.com/foo: foo
      nodeSelector: whatever
      priorityClassName: whatever
      securityContext:
        something: whatever
      terminationGracePeriodSeconds: 111
      tolerations:
        fakeNonsense: idk the actual format
    replicaCount: 3
  enabled: true
  gatewayDiscovery:
    enabled: true
    generateAdminApiService: true
  ingressClass: kong
  konnect:
    runtimeGroupID: abc123
  rbac:
    create: true
  watchNamespaces:
  - default
  - other
